README.md

```diff
@@ -1,41 +1,44 @@
-# SDK Demo
+# Sample dApp Template
 
-This is a simple app for integrating and testing Arcana's login and storage SDKs.
-Use this app to implement and test SSO (Oauth) based login and file based transactions like upload, download, delete and share
+This is a simple Vue application template. Use it to build a full-fledged dApp integrated with Arcana Auth and Storage SDKs. Learn how to enable user onboarding and data privacy features in dApp.
+
+Refer to [Arcana Network documentation](https://docs.beta.arcana.network/docs/starter-app) for step by step instructions on how to use this template and integrate with Arcana Auth and Storage SDKs for enabling user onboarding and data privacy features.
 
 ## Quick Start
 
 - Prerequisites:
 
-  - You'll need an up to date LTS versions of **`node.js`** and **`npm`** installed in your system for this code to work.
-  - You'll need a smart contract address. Get one at https://developer.arcana.network by creating an app.
-  - You'll need client id of one of the following for login: Google, Twitch, Discord, Reddit, GitHub. You'll need to add this client id in the app that you created in [Arcana's Developer Dashboard](https://developer.arcana.network)
+  - You'll need latest LTS versions of **`node.js`** and **`npm`** installed in your system for this code to work.
+  - You'll need to register and configure your dApp using [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network). Configure social authentication mechanism, storage region, storage limits and other settings using the dashboard.
+  - You'll need client ID or other social authentication identifiers for the selected user onboarding mechanisms: Google, Twitter, Twitch, Discord, Reddit, GitHub. You'll need to specify the client ID or other identifier created through social authentication provider in [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network)
```

Comment:
```suggestion
You'll need client ID for the selected user onboarding mechanism in case of this dApp: Google. You'll need to specify the client ID or other identifier created through social authentication provider in [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network)